dts/bindings/pinctrl/microchip,pic32cxsg-pinctrl.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2024 Microchip
```

Comment:
if you are coping and renaming, keep original copyrights.

This should be in the previous commit.